Update the README.md file to describe the repo.

aliu917 opened the pull request:
I updated the readme as the prompts instructed.

aliu917 committed:
Update the README.md file to describe the repo.

aliu917 committed:
Did all the required stuff.

aliu917 committed:
Added a github link

aliu917 committed:
Added some sectional structure.

dibenede commented on portfolio/src/main/webapp/script.js:
```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
```
Rename greeting -> fact here and below

dibenede reviewed (changes requested):
I recommend keeping the commit message as descriptive as possible. Git typically uses the first 80 characters before a newline as the brief message that will show up in the short commit log. You can then expand on it for the extended history view. So, for this change, I would do something like:

Add and format data about me (I think that's less than 80)

* Add a summary about me.
* Make information about my hobbies available on click.
* Add basic formatting.

dibenede commented on portfolio/src/main/webapp/index.html:
```diff
@@ -10,14 +10,27 @@
-      <img border="0" alt="Github" src="images/github.png" width="50" height="50">
+        <img border="0" alt="Github" src="images/github.png" width="50" height="50">
       </a>
-      <p></p>
```
optional: I think you can just do ```<p />``` for things like this

aliu917 committed:
Changed the design and formatting

aliu917 committed:
Finished the project section

aliu917 commented:
A question about the commit messages: are you saying that each commit should contain the change I made for that commit in the first line and then bullet points for all the previous commit messages I made? So like for my most recent commit about adding a project section, it would be:
Finished the project section

- changed design and formatting
- add a summary about me
- add basic formatting
- make information about hobbies available on click

aliu917 committed:
Redesigned the random section and also fixed greetings to facts

dibenede commented:
> A question about the commit messages: are you saying that each commit should contain the change I made for that commit in the first line and then bullet points for all the previous commit messages I made? So like for my most raecent commit about adding a project section, it would be:
> Finished the project section
> 
> * changed design and formatting
> * add a summary about me
> * add basic formatting
> * make information about hobbies available on click

No, sorry. I may have put my comment in a weird spot. Basically your first line should be a summary. If you have a small commit, the summary may be sufficient to describe what you did and you can call it good.

If you have a large commit, then it's a good idea to list out what's included in it. Sorry for the confusion.

aliu917 committed:
Added a hello world page:
- created another html page linked from the original index html
- created a greeting container that holds "hello world" phrase
- implemented next greeting generation for recurring greeting change
- added a link back to the main page when clicking on my name

aliu917 committed:
Made a minor change to the text
